plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftConfig.java

```diff
@@ -40,4 +43,30 @@ public RedshiftConfig setFetchSize(Integer fetchSize)
         this.fetchSize = fetchSize;
         return this;
     }
+
+    public Optional<String> getUnloadLocation()
+    {
+        return Optional.ofNullable(unloadLocation);
+    }
+
+    @Config("redshift.unload-location")
+    @ConfigDescription("A writeable location in Amazon S3, to be used for unloading Redshift query results")
+    public RedshiftConfig setUnloadLocation(String unloadLocation)
+    {
+        this.unloadLocation = unloadLocation;
+        return this;
+    }
+
+    public Optional<String> getIamRole()
+    {
+        return Optional.ofNullable(iamRole);
+    }
+
+    @Config("redshift.iam-role")
```

Comment:
We should disallow setting this config property if `redshift.unload-location` is an empty.

Reply:
Maybe rename to `redshift.unload-iam-role`

Reply:
should we allow reusing auth configs from RedshiftAwsCredentialsConfig ?

Reply:
renamed.